src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -174,6 +169,103 @@ const MissingData = styled.div`
   gap: 20px;
 `
 
+export const AboutContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  padding: 24px 0px;
+`
+
+const TokenDescriptionContainer = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 100%;
+  max-height: fit-content;
+`
+
+const TruncateDescriptionButton = styled.div`
+  color: ${({ theme }) => theme.textSecondary};
+  font-weight: 400;
+  font-size: 14px;
+  padding: 12px 0px;
+
+  &:hover,
+  &:focus {
+    color: ${({ theme }) => darken(0.1, theme.textSecondary)};
+    cursor: pointer;
+  }
+`
+
+const TRUNCATE_WORDCOUNT = 60
+
+type TokenDetailData = {
+  description: string | null
+  homepageUrl: string | null
+  twitterName: string | null
+}
+
+export function AboutSection({ address, tokenDetailData }: { address: string; tokenDetailData: TokenDetailData }) {
+  const [isDescriptionTruncated, setIsDescriptionTruncated] = useState(true)
+
+  if (!tokenDetailData || !tokenDetailData.description) {
+    return (
+      <AboutContainer>
+        <AboutHeader>
+          <Trans>About</Trans>
+        </AboutHeader>
+        <NoInfoAvailable>
+          <Trans>No token information available</Trans>
+        </NoInfoAvailable>
+        <ResourcesContainer>
+          <Resource name={'Etherscan'} link={`https://etherscan.io/address/${address}`} />
+          <Resource name={'Protocol Info'} link={`https://info.uniswap.org/#/tokens/${address}`} />
+        </ResourcesContainer>
+      </AboutContainer>
+    )
+  } else {
+    const tokenDescriptionArray = tokenDetailData.description.split(' ')
+    const shouldTruncate = tokenDescriptionArray.length > TRUNCATE_WORDCOUNT
+    const tokenDescriptionTruncated = shouldTruncate
+      ? tokenDescriptionArray.slice(0, TRUNCATE_WORDCOUNT - 1).join(' ') + '...'
```

Comment:
`${tokenDescriptionArray.slice(0, TRUNCATE_WORDCOUNT - 1).join(' ')}...`

Reply:
updated to character count due to fred instructions to include paragraph breaks